### PR TITLE
Adding .update() to docs

### PR DESCRIPTION
**Pull request by aspiiire**

### CODE REVIEW

File: website/docs/concepts/reading.mdx [hunk 1]
```diff
@@ -275,6 +275,52 @@ If we used `ref.read` here, our button would still use the previous
 `StateController` instance, which was disposed and should no-longer be used.  
 Whereas using `ref.watch` correctly rebuilt the button to use the new `StateController`.
 
+### Using .update() to update previous state esily
```
Comment (rrousselGit) [inline website/docs/concepts/reading.mdx:278]: I don't like this being here, since:
- it's not about "reading"
- it's specific to `StateProvider`

**Comment by rrousselGit** [comment]
Thanks! Although I'm not sure that's the correct place to put such doc

**Comment by aspiiire** [comment]
> Thanks! Although I'm not sure that's the correct place to put such doc

I thought about it and at the end decided to put it there cause was more a ref.read specific scenario. You are right, it is not about reading! The other pages I think was too way off, do you suggest any place in which I can put it?

**Comment by rrousselGit** [comment]
I think the place where it would make the most sense is on a page dedicated to StateProvider

File: website/docs/concepts/reading.mdx (hunk 1 above)
Comment (GitGud31) [reply website/docs/concepts/reading.mdx:278]: @rrousselGit  There's a typo in **esily** (easily) to be fixed before merging

**Comment by aspiiire** [comment]
> 

What do you think? better?

**Comment by rrousselGit** [comment]
Hum no, it's not the correct page either.

Maybe add that to the dartdoc of StateProvider. https://pub.dev/documentation/riverpod/latest/riverpod/StateProvider-class.html

**Comment by aspiiire** [comment]
> Hum no, it's not the correct page either.
> 
> Maybe add that to the dartdoc of StateProvider. https://pub.dev/documentation/riverpod/latest/riverpod/StateProvider-class.html

Ok but I have to take a little bit of time since I have no idea on how to contribute to dartdoc.

**Comment by rrousselGit** [comment]
I'll try to make a page for all providers this week-end. Then I'll move this to the Stateprovider page

**Comment by aspiiire** [comment]
> I'll try to make a page for all providers this week-end. Then I'll move this to the Stateprovider page

I can close the pull request?

**Comment by rrousselGit** [comment]
No need. I'll take care of things from there and rebase/merge it when I'm done with other things

**Comment by rrousselGit** [comment]
Done, thanks for your contribution!

**Pull request merged**